scripts/migrations/002-remove-webview-flags.js

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env node
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+'use strict';
+const fs = require('fs');
+const path = require('path');
+const { platform } = require('os');
+
+/** Determines if the OS is Windows */
+const IS_WINDOWS = platform() === 'win32';
+
+const removeWebViewFlags = (key, value) => {
+  if (key === "__compat") {
+    if (value.support.webview_android !== undefined) {
+      if (Array.isArray(value.support.webview_android)) {
+        var result = [];
+        for (var i = 0; i < value.support.webview_android.length; i++) {
+          if (value.support.webview_android[i].flags === undefined) {
+            result.push(value.support.webview_android[i]);
+          }
+        }
+
+        value.support.webview_android = result.length > 1 ? result : result[0];
+      } else if (value.support.webview_android.flags !== undefined) {
+        value.support.webview_android = {"version_added": false};
+      }
+    }
+  }
+  return value;
+};
+
+ /**
+  * @param {Promise<void>} filename
+  */
+const fixWebViewFlags = (filename) => {
+  let actual   = fs.readFileSync(filename, 'utf-8').trim();
```

Comment:
Admittedly, a small thing, but I like using what meager guardrails JS gives us 😆

```suggestion
  const actual   = fs.readFileSync(filename, 'utf-8').trim();
```